From fe1781d10dd8734af21dbea8c070069829456c03 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Thu, 12 Nov 2020 13:05:05 +0100
Subject: [PATCH] meson: allow oomd to be enabled even in release mode

A distro (Fedora in particular) may want to enable oomd in a unstable
branch for testing, even though the package as a whole is compiled in release
mode. Let's emit a warning but otherwise allow this.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 7996bc2c65..43e328f5a6 100644
--- a/meson.build
+++ b/meson.build
@@ -1419,7 +1419,7 @@ if have == 'auto'
 else
         have = have == 'true'
         if have and get_option('mode') != 'developer'
-                error('oomd is not available in release mode (yet)')
+                warning('oomd is not ready for release mode (yet)')
         endif
 endif
 conf.set10('ENABLE_OOMD', have)
